seed/data/dev/00700_workload.js

```diff
@@ -2,13 +2,13 @@ var tableName = 'workload'
 
 const workloadRow = {
   workload_owner_id: 1,
-  total_cases: 25,
+  total_cases: 63,
   monthly_sdrs: 1,
   sdr_due_next_30_days: 2,
   sdr_conversions_last_30_days: 1,
-  total_community_cases: 10,
-  total_custody_cases: 7,
-  total_license_cases: 8,
+  total_community_cases: 21,
```

Comment:
These numbers match the total per workload in the tiers table? How much of a pain would it be to create different values for each location?

Reply:
Added different tiers values for each location but the totals are all still the same

Reply:
Tiers data is pulled from extract, then this data is used to populate workload

Reply:
Why pick data where the totals are still the same? You use the same data but just shuffle the tiers? 😛

Reply:
Yeah rather than creating whole new tiers data for each one I've swapped some of the tiers, I've now also added some extra cases to some of the tiers